doc/src/guides/solving/solve-matrix-equation.md

```diff
@@ -0,0 +1,126 @@
+# Solve a Matrix Equation
+
+Use SymPy to solve a matrix (linear) equation. For example, solving $
+\left[\begin{array}{cc} c & d\\1 & -e\end{array}\right] \left[\begin{array}{cc}
+x\\y\end{array}\right] = \left[\begin{array}{cc} 2\\0\end{array}\right] $ yields
+$ \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc}
+\frac{2e}{ce+d}\\\frac{2}{ce+d}\end{array}\right]$.
+
+## Alternatives to Consider
+- Solving a matrix equation is equivalent to solving a system of linear
+  equations, so if you prefer you can use *link to solve a system of equations
+  once published*
+- If you formulated your problem as a system of linear equations, and want to
+  convert it to matrix form, you can use {func}`~.linear_eq_to_matrix` and then
+  follow the procedures in this guide.
```

Comment:
@asmeurer I found the function to convert a set of linear equations to matrix-equation form, [linear_eq_to_matrix](https://docs.sympy.org/latest/modules/solvers/solveset.html?highlight=linear_eq_to_matrix#linear-eq-to-matrix). I did not find a function to do the reverse; if there is one, please let me know.